### PR TITLE
Celements 4 Preparation

### PR DESCRIPTION
**Pull request by msladek**
https://synjira.atlassian.net/browse/CELDEV-811

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -220,7 +218,7 @@
     <dependency>
       <groupId>com.google.code.findbugs</groupId>
       <artifactId>jsr305</artifactId>
-      <version>3.0.1</version>
+      <version>3.0.2</version>
```
Comment (fpichler) [inline pom.xml:222]: Can we change to SpotBugs as findbugs is not maintained anymore? see https://gleclaire.github.io/findbugs-maven-plugin/
and https://spotbugs.readthedocs.io/en/latest/maven.html

**Review by fpichler — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (msladek) [reply pom.xml:222]: see answer on base-pom

Comment (fpichler) [reply pom.xml:222]: please add comment for TODO: https://synjira.atlassian.net/browse/CELDEV-813

**Review by fpichler — CHANGES_REQUESTED** [review]
please add comment for TODO: https://synjira.atlassian.net/browse/CELDEV-813

File: pom.xml [hunk 2]
```diff
@@ -220,7 +218,7 @@
     <dependency>
```
Comment (fpichler) [inline pom.xml:219]:
```suggestion
<!-- TODO: https://synjira.atlassian.net/browse/CELDEV-813 -->
    <dependency>
```

**Review by fpichler — CHANGES_REQUESTED** [review]

**Review by fpichler — APPROVED** [review]

**Pull request merged**